### DISCUSSION TITLE
module.unsafeCache and cache option

### DISCUSSION THREAD

**Discussion by jianxingyao** [webpack/webpack#18953] [General]
Do module.unsafeCache and cache options only need to be set, and if they are set at the same time, will it speed up the build performance?
Hope someone can say the difference between unsafeCache and cache Thanks!

**Comment by alexander-akait** [comment]
We use this logic - https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L627, so if cache enabled you don't need to set this option

**Reply by jianxingyao** [reply]
Thanks, solved my doubts
